composer.json

```diff
@@ -181,7 +181,7 @@
       "WordPressProject\\composer\\ScriptHandler::createRequiredFiles"
     ],
     "code-sniff": [
-      "./vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs,standards/RDA && ./vendor/bin/phpcs --standard=RDA --extensions=php --runtime-set ignore_warnings_on_exit true --ignore=./web/wp-content/plugins/a3-lazy-load/*,./web/wp-content/plugins/advanced-custom-fields-pro/*,./web/wp-content/plugins/autoptimize/*,./web/wp-content/plugins/co-authors-plus/*,./web/wp-content/plugins/customize-snapshots/*,./web/wp-content/plugins/easyazon/*,./web/wp-content/plugins/official-facebook-pixel/*,./web/wp-content/plugins/lh-hsts/*,./web/wp-content/plugins/marquee/legacy/*,./web/wp-content/plugins/members/*,./web/wp-content/plugins/pantheon-advanced-page-cache/*,./web/wp-content/plugins/radio-buttons-for-taxonomies/*,./web/wp-content/plugins/redirection/*,./web/wp-content/plugins/regenerate-thumbnails/*,./web/wp-content/plugins/user-switching/*,./web/wp-content/plugins/wordpress-seo-premium/*,./web/wp-content/plugins/wp-cfm/*,./web/wp-content/plugins/wpmudev-updates/*,./web/wp-content/plugins/wp-native-php-sessions/*,./web/wp-content/plugins/wp-redis/*,./web/wp-content/plugins/wp-smush-pro/*,./web/wp-content/plugins/wp-security-audit-log/*,./web/wp-content/plugins/wp-migrate-db-pro*,./web/wp-content/mu-plugins,./web/wp-content/themes/tw*,./web/wp-content/uploads ./web/wp-content -n"
+      "./vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs,standards/RDA && ./vendor/bin/phpcs --standard=RDA --extensions=php --runtime-set ignore_warnings_on_exit true --ignore=./web/wp-content/plugins/a3-lazy-load/*,./web/wp-content/plugins/advanced-custom-fields-pro/*,./web/wp-content/plugins/autoptimize/*,./web/wp-content/plugins/co-authors-plus/*,./web/wp-content/plugins/customize-snapshots/*,./web/wp-content/plugins/easyazon/*,./web/wp-content/plugins/official-facebook-pixel/*,./web/wp-content/plugins/lh-hsts/*,./web/wp-content/plugins/marquee/legacy/*,./web/wp-content/plugins/members/*,./web/wp-content/plugins/pantheon-advanced-page-cache/*,./web/wp-content/plugins/radio-buttons-for-taxonomies/*,./web/wp-content/plugins/redirection/*,./web/wp-content/plugins/regenerate-thumbnails/*,./web/wp-content/plugins/user-switching/*,./web/wp-content/plugins/tmbi-field-validation/vendor/*,./web/wp-content/plugins/wordpress-seo-premium/*,./web/wp-content/plugins/wp-cfm/*,./web/wp-content/plugins/wpmudev-updates/*,./web/wp-content/plugins/wp-native-php-sessions/*,./web/wp-content/plugins/wp-redis/*,./web/wp-content/plugins/wp-smush-pro/*,./web/wp-content/plugins/wp-security-audit-log/*,./web/wp-content/plugins/wp-migrate-db-pro*,./web/wp-content/mu-plugins,./web/wp-content/themes/tw*,./web/wp-content/uploads ./web/wp-content -n"
```

Comment:
It'd be nice to resolve this at some point. Right now every time we add a plugin we have to change the composer.json for validation OR the .gitignore OR both. It's not ideal and causes conflicts. We need a ticket to address this.